lib/membrane_rtc_engine/engine.ex

```diff
@@ -347,9 +337,9 @@ defmodule Membrane.RTC.Engine do
   @doc """
   Removes peer from RTC Engine.
   """
-  @spec remove_peer(rtc_engine :: pid(), peer_id :: any()) :: :ok
-  def remove_peer(rtc_engine, peer_id) do
-    send(rtc_engine, {:remove_peer, peer_id})
+  @spec remove_peer(rtc_engine :: pid(), peer_id :: any(), reason :: String.t()) :: :ok
+  def remove_peer(rtc_engine, peer_id, reason \\ "normal") do
```

Comment:
```suggestion
  def remove_peer(rtc_engine, peer_id, reason \\ nil) do
```
We should fix typespec too.